backend/src/utils/get-logbook-type.js

```diff
@@ -0,0 +1,11 @@
+export default async function getLogbookType(logbookID, supabase) {
+    try {
+        const { data, error } = await supabase.from("logbooks").select().eq("id", logbookID);
+        if (error) {
+            throw new Error(error.message);
+        }
+        return data[0]['type'];
```

Comment:
If we cannot find the logbook, data[0] will be undefined. In this case, we should return a more specific error regarding not being able to find a logbook with the specified logbookId, and pass that to the service